YapDatabase/Utilities/YapDatabaseCryptoUtils.h

```diff
@@ -8,11 +8,10 @@ NS_ASSUME_NONNULL_BEGIN
 
 extern const NSUInteger kSqliteHeaderLength;
 extern const NSUInteger kSQLCipherSaltLength;
-extern const NSUInteger kSQLCipherDerivedKeyLength;
```

Comment:
Why rename this?  I'm deliberately trying to differentiate it from the "database password/key". 

Consider `YapDatabaseOptions`: it has a property named `cipherKeyBlock` of type `YapDatabaseCipherKeyBlock`.  This length does _NOT_ apply to that key.

In fact, I should have added a comment here emphasizing this distinction.

Reply:
Reverted.

In a previous version of this PR, this constant was used externally and in the context of generating (not deriving) a raw key I was naming to be consistent between "password" vs "key" (not all keys are derived, but they are all this length). 

Since I pulled all that stuff out, and we're once again *only* using this constant within the context of deriving keys, I'm fine with the original name.